src/analyze.cpp

```diff
@@ -1058,7 +1058,10 @@ ZigType *get_ptr_to_stack_trace_type(CodeGen *g) {
     if (g->stack_trace_type == nullptr) {
         ConstExprValue *stack_trace_type_val = get_builtin_value(g, "StackTrace");
         assert(stack_trace_type_val->type->id == ZigTypeIdMetaType);
+
         g->stack_trace_type = stack_trace_type_val->data.x_type;
+        assertNoError(ensure_complete_type(g, g->stack_trace_type));
```

Comment:
I think this could be potentially problematic - get_pointer_to_type only needs to know that the type is not zero bits. (you can see that in the assertion failed message). `ensure_complete_type` is deprecated - it's the same as `type_resolve(type, ResolveStatusSizeKnown)`. So I think this should be `type_resolve(g->stack_trace_type, ResolveStatusZeroBitsKnown)`.

The `assertNoError` would be a problem stack_trace_type were a user type, because then a compile error in that type would make this assertion trip. So this is ok as long as we know that a compile error in the StackTrace builtin type will result in a compiler assertion. (I do think that is OK.)